src/css/styles/_c-banner.scss

```diff
@@ -0,0 +1,82 @@
+.c-banner {
+  position: relative;
+  width: 100%;
+  min-height: 568px;
+  max-height: 1440px;
+  margin: 0 auto;
+  margin-bottom: 6rem;
+  background: url("../img/c-banner__bg.jpg") no-repeat 40% 60%;
+  background-size: auto 130%;
+
+  &__title {
+    margin-bottom: 3rem;
+    font-size: 4rem;
+    line-height: 120%;
+    text-shadow: 0 0 10px $col-bg;
+  }
+
+  &__txt {
+    margin-bottom: 4rem;
+    font-size: 1.4rem;
+    line-height: 120%;
```

Comment:
![image](https://user-images.githubusercontent.com/789841/50001833-650bfc00-ff9e-11e8-9a8a-64bdd527876f.png)

- [ ] 120% vs 150%

Reply:
Yup, looks better. For mobiles it's 140%, above xs it's 150%.